BUILDING.md

```diff
@@ -146,7 +146,11 @@ platforms. This is true regardless of entries in the table below.
   systems. Issues that only reproduce on WSL should be reported in the
   [WSL issue tracker](https://github.com/Microsoft/WSL/issues). Running the
   Windows binary (`node.exe`) in WSL is not recommended. It will not work
-  without workarounds such as stdio redirection.
+  without workarounds such as stdio redirection. 
+  If you run into a `No module named 'distutils.spawn'` error when executing
+  `./configure`, please try `python3 -m pip install --upgrade setuptools` or
+  `sudo apt install python3-distutils -y`.
+  For more information, see https://github.com/nodejs/node/issues/30189.
```

Comment:
https://github.com/nodejs/code-and-learn/pull/99 suggests this is not WSL specific.

Reply:
Yes.  That is why I changed the title of this PR

Reply:
Right, but this change is currently in the WSL notes. Maybe it should be moved to either the section about Python 2 and 3 or moved into its own section.